### ISSUE TITLE
Please do not call it "install"

### ISSUE THREAD

**Issue by probonopd** [azubieta/appimage-user-tools#21]
AppImages do not need to be installed. They are just downloaded and made executable, and optionally integrated into the system.

Please do not use the verb "install". Maybe "get".

**Comment by TheAssassin** [comment]
The term `install` was chosen as the tool does not just download the files but also puts them into `~/Applications` and integrates them. `get` doesn't cover that. Unless there's a more suitable term, `install` is closest to what's actually happening.

**Comment by probonopd** [comment]
Only that AppImages do not need to be installed. That is one of their main selling points. 

In fact I made AppImage to not need the concept of "installation", repositories, and package managers.

**Comment by TheAssassin** [comment]
Doesn't mean that these things can't be built on top. This tool's purpose is "be a client for a store that offers AppImages". The tool "installs" these and manages them.

If you want to change the nomenclature, please make a few suggestions. We haven't come up with anything better than what is used right now yet.

**Comment by probonopd** [comment]
Above I had suggested to use the verb "get". Many users will know it from apt.

**Comment by TheAssassin** [comment]
You know that the command to "install" packages is `apt install`, not `apt get` (or `apt-get get`)?

**Comment by probonopd** [comment]
Yes, because those commands actually do install the software into the system.

**Comment by TheAssassin** [comment]
So how is this tool differently? Just because "desktop integration" doesn't use the term "install", it doesn't mean that files aren't installed into the system. Also, when tools begin to add restrictions like "move into specific directory", they exceed the term "get". And `app integrate XYZ` (where `XYZ` is the AppImage to download and integrate) seems misleading.

**Comment by star-buck** [comment]
"install" is exactly what it does as a mere get.
Get implies only download, install implies download and proper system integration like an icon in the menu etc.
The mere techncial fact that the app doesnt need to be "extracted" to be integrated is totally irrelevant for the user, ALL users of ALL operating system know that install = properly integrated end result of a software regardless its file format.
